### PR TITLE
Implement replacer for CHANGELOG file

### PR DESCRIPTION
**Pull request by Andre601**
## Checks

I have...

- [x] read and understood the [Contributing Guidelines][contributing]
- [x] Updated any nessecary files such as the README.md and/or CHANGELOG.md.

## Type of Pull request

- [ ] **Minor Change**  
  This Pull request doesn't break existing configuration.
- [ ] **Major Change**  
  This Pull request will break existing configuration.
- [ ] **Bug fix**  
  This Pull request will fix a (critical) bug.
- [ ] **Documentation**  
  This Pull request only changes documentation (README.md, CHANGELOG.md, etc.)
- [x] **Other:** `Changelog handling` 

## Description

Implements the Readme-Replacer action to automatically replace the `{tba}` placeholder in the `CHANGELOG.md` when a release is published.
This saves us some work.

Closes #

[contributing]: https://github.com/Readme-Workflows/.github/blob/main/.github/CONTRIBUTING.md

### CODE REVIEW

File: .github/workflows/update_changelog.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  build:
+    runs-on: ubuntu-latest
+    name: Replace {tba} placeholder
+    steps:
+    - uses: actions/checkout@v2.3.4
+    - uses: Readme-Workflows/readme-replacer@v1.0.1
+        env:
+          GITHUB_TOKEN: {{ secrets.GITHUB_TOKEN }}
+        with:
+          # Those are all default values and only are shown for demonstration
```
Comment (PuneetGopinath) [inline .github/workflows/update_changelog.yml:18]:
```suggestion
```

**Review by PuneetGopinath — COMMENTED** [review]
LGTM. But that comment isn't required.

**Comment by abhijoshi2k** [comment]
The issue I observed on readme-replacers repo is that the action fails on release. However, it is successful if run manually. I don't understand why.

**Comment by PuneetGopinath** [comment]
> The issue I observed on readme-replacers repo is that the action fails on release. However, it is successful if run manually. I don't understand why.

It doesn't fail, but GitHub actions never runs it on time no(on release)
Very less chances to run automatically. Problem with gh actions.

**Comment by abhijoshi2k** [comment]
It does trigger on time. But it fails. So I had to remove the trigger

**Review by abhijoshi2k — CHANGES_REQUESTED** [review]
Please add `workflow_dispatch` trigger too. In case it fails on release (which it did on `readme-replacer` repo.

**Comment by Andre601** [comment]
I turn this into draft again because there seems to be a random bug(?) probably related to drafting releases where the action is triggered constantly through commits...
Need to ask GitHub community about this.

**Comment by Andre601** [comment]
I think I found the issue, which was a syntax error on my end...

**Review by abhijoshi2k — APPROVED** [review]

**Pull request merged**